www/codePush.ts

```diff
@@ -293,7 +294,7 @@ class CodePush implements CodePushCordovaPlugin {
             CodePushUtil.copyUnassignedMembers(defaultOptions, syncOptions);
         }
 
-        window.codePush.notifyApplicationReady();
+        window.codePush.notifyApplicationReady(null, null, syncOptions.deploymentKey);
```

Comment:
Can't we make the deploymentKey the first param? Then we wouldn't have to do `null, null`. Seems more idiomatic for node style async callback functions too.